packages/playwright-ct-angular/index.d.ts

```diff
@@ -15,39 +15,31 @@
  */
 
 import type {
-  TestType,
+  Locator,
   PlaywrightTestArgs,
-  PlaywrightTestConfig as BasePlaywrightTestConfig,
   PlaywrightTestOptions,
   PlaywrightWorkerArgs,
   PlaywrightWorkerOptions,
-  Locator,
+  TestType,
```

Comment:
`TestType` should be imported from `@playwright/experimental-ct-core` _(and `defineConfig` should be exported `@playwright/experimental-ct-core`)_

Reply:
Nice catch!
I just made it align with other ct packages as much as possible.